api/_db.js

```diff
@@ -58,8 +61,7 @@ const fetchAllData = async (client) => {
 
 // raw input from the sheet => valueof CODE
 const normalizeCertificate = (rawCertificate) => {
-	assert.equal(typeof rawCertificate, 'string', '"rawCertificate" required');
-
+	rawCertificate = rawCertificate ?? '';
```

Comment:
Don't we want to differentiate between someone who is in the DB but has no certification and someone who's not even in the DB?

Reply:
Doesn't the early return upon no result by id take care of that?

Reply:
I guess you changed it because of the google api having `null` or `undefined` as the value of the empty cell?
In that case we'd like to have a default value, but still throw if we've done a coding error. Turned that back into a little bit more secure solution keeping that in mind.